gui/src/components/maintenance/cortx-software.vue

```diff
@@ -136,16 +136,40 @@ export default class CortxHotfix extends Vue {
     isDirty: false,
     isValid: false,
   };
-
+  public systemStatus: boolean = true;
   public async mounted() {
+    await this.getSyetmStatus();
     await this.getLastUpgradeStatus();
   }
-
+ public async getSyetmStatus() {
+    this.$store.dispatch(
+      "systemConfig/showLoader",
+      "checking service status..."
+    );
+    try {
+      const res: any = await Api.getAll(apiRegister.system_status);
+      this.$store.dispatch("systemConfig/hideLoader");
+    } catch (error) {
+      this.$data.systemStatus = false;
+      let errorMessage = "please check service status";
+      if (error && error.error) {
+        errorMessage = error.error.message;
+      }
+      throw {
+        error: {
+          message: errorMessage
+        }
+      };
+    } finally {
+      this.$store.dispatch("systemConfig/hideLoader");
+    }
+  }
   public async getLastUpgradeStatus() {
     this.$store.dispatch(
       "systemConfig/showLoader",
       "Fetching last update status..."
     );
+     this.getSyetmStatus();
     const res: any = await Api.getAll(apiRegister.hotfix_status);
     this.lastUpgradeStatus = res && res.data ? res.data : null;
     if (this.lastUpgradeStatus.status === "uploaded") {
```

Comment:
This should be an enum instead of plain, text.

Reply:
can we add those changes in separate PR?